client/src/tpu_client.rs

```diff
@@ -55,27 +62,12 @@ impl TpuClient {
         self.tpu_client.try_send_wire_transaction(wire_transaction)
     }
 
-    /// Create a new client that disconnects when dropped
-    pub fn new(
```

Comment:
I think this is a pub function used down-stream and externally for some time -- we cannot remove it

Reply:
Reverted this back.

To be clear, this function adds dependency on `client/ConnectionCache`.
`client/ConnectionCache` depends on both `QUIC` and `UDP` implementation.
So, unless this method is removed from the interface, `client/TpuClient` will depend on both implementations.

This :point_up: is the status quo on _the current master_ branch; _not_ a consequence of this commit.